ios/CovidShield/ExposureNotification.m

```diff
@@ -9,30 +9,9 @@
 #import <React/RCTConvert.h>
 
 
-@interface ENManagerMock : ENManager
-
-@end
-
-
-@implementation ENManagerMock
-
-- (void)getDiagnosisKeysWithCompletionHandler:(ENGetDiagnosisKeysHandler)completionHandler {
-  ENTemporaryExposureKey *key = ENTemporaryExposureKey.new;
-  key.keyData = [@"1234567812345678" dataUsingEncoding:NSUTF8StringEncoding];
-  key.rollingStartNumber = 123;
-  key.transmissionRiskLevel = 5;
-  
-  NSArray *keys = @[key];
-  completionHandler(keys, nil);
-}
-
-@end
-
-
-typedef ENManagerMock ENManagerImplementation ;
```

Comment:
👍

Reply:
(a mock might be worth adding back in at a later date once we have a better idea of what the framework does)

Reply:
yeah but mock data I tried to come up with doesn't make sense and causes errors when attempt to use by both framework AND backend.